cmd/crc/cmd/root.go

```diff
@@ -37,14 +35,14 @@ func init() {
 	config.InitViper()
 	setConfigDefaults()
 	rootCmd.AddCommand(cmdConfig.ConfigCmd)
-	rootCmd.PersistentFlags().StringVar(&logLevel, "log-level", "info", "log level (e.g. \"debug | info | warn | error\")")
+	rootCmd.PersistentFlags().StringVar(&constants.LogLevel, "log-level", "info", "log level (e.g. \"debug | info | warn | error\")")
```

Comment:
I think we should be clearer: `constants.DefaultLogLevel` would make more sense here.

Reply:
... oh wait... you are setting the value here INTO the constant?!????

Reply:
```
rootCmd.PersistentFlags().StringVar(&logging.LogLevel, "log-level", constants.DefaultLogLevel, "log level (e.g. \"debug | info | warn | error\")")
```